torch/utils/cpp_extension.py

```diff
@@ -571,13 +571,31 @@ def _append_sycl_std_if_no_std_present(cflags):
         cflags.append('-sycl-std=2020')
 
 
-def _wrap_sycl_host_flags(cflags):
+def _wrap_sycl_host_flags(host_cflags):
```

Comment:
To be honest I don't much care, but what was the reason to rename `cflags` to `host_cflags`? To me that's a user preference naming change and I would stick with the existing naming to minimize the code difference and make it easier to revie.